java-security-test/src/test/resources/client_token.json

```diff
@@ -0,0 +1,8 @@
+{
+  "header": {
```

Comment:
Empty header is OK. If it is left out we have not defined what happens. Right now it would be an NullPointer which is bad. What should happen? Also just ignore it?

Reply:
Yes, no header should behave the same like "empty" one to make it a little bit more robust.